manuscript/portal_iterative_forecasting_ms.Rmd

```diff
@@ -41,32 +41,32 @@ Key-words: forecasting, prediction, mammals, iterative forecasting, Portal Proje
 
```

Comment:
re: end of abstract (it won't let me comment on the exact line, sorry!)
"system" is used a lot in point 4, and in general the "forecasting systems" vs "natural systems" helps distinguish between the two uses. I'd suggest continuing it through to the end, so that "build these systems" would be "build forecasting systems" or something akin.